mkosi/state.py

```diff
@@ -1,14 +1,17 @@
 # SPDX-License-Identifier: LGPL-2.1+
 
 from pathlib import Path
+from typing import Optional
 
 from mkosi.config import MkosiArgs, MkosiConfig
+from mkosi.log import die
 from mkosi.tree import make_tree
 from mkosi.util import umask
 
 
 class MkosiState:
     """State related properties."""
+    source_date_epoch: Optional[int] = None
```

Comment:
Why make this a class variable instead of an instance variable?

Reply:
just unsure how to use type hints properly :sweat_smile: